sdk/communication/Azure.Communication.JobRouter/tests/Infrastructure/RouterLiveTestBase.cs

```diff
@@ -61,7 +61,14 @@ protected void AssertQueueResponseIsEqual(Response<UpsertQueueResponse> upsertQu
             Assert.AreEqual(distributionPolicyId, response.DistributionPolicyId);
             if (queueLabels != default)
             {
-                Assert.AreEqual(queueLabels, response.Labels);
+                var someting = queueLabels.ToDictionary(k => k.Key, k => k.Value);
+                Assert.True(response.Labels.ContainsKey("Id"));
+                Assert.AreEqual(queueId, response.Labels["Id"]);
+
+                // copy label collection to a new dictionary - do not implicitly modify response object within function
+                var responseLabelCollection =
```

Comment:
Would it make sense to have a reusable extension method or something for label comparison for tests?